Gemfile.lock

```diff
@@ -143,7 +143,6 @@ PATH
 
 GEM
   remote: https://rubygems.org/
-  remote: https://enterprise.contribsys.com/
```

Comment:
don't check in changes to the gemfile that remove `remote: https://enterprise.contribsys.com/` and related gems

Reply:
Fixed! Please re-review. Thanks!